README.md

```diff
@@ -1,3 +1,8 @@
+## Microsoft Open Source Code of Conduct
```

Comment:
this project is under  the .NET Foundation Code of Conduct which is called out below. see https://opensource.microsoft.com/codeofconduct/faq/#alreadyhave